### DISCUSSION TITLE
Exonérations ZRR

### DISCUSSION THREAD

**Discussion by juliencarnot** [betagouv/mon-entreprise#2603] [Idées]
Je viens de redécouvrir les [exonérations pour les embauches en ZRR](https://www.afecreation.fr/pid538/exoneration-sociales-en-zrr.html), qui sont très importantes mais qui n'apparaissent pas dans le simulateur, qui ne demande pas la commune d'embauche et n'applique donc pas les règles en découlant.

Ce qui m'amène à une autre interrogation: avez-vous un indicateur sur la couverture du simulateur sur toutes les règles applicables en droit social, niches sociales et exceptions comprises? Dans ce cas, serait-il possible de l'afficher plus clairement, par exemple en complétant le disclaimer présent en pied de page, en ajoutant qu'outre les conventions et accords collectifs, x% des règles manquent encore au simulateur?

**Comment by laem** [comment]
Oui notre couverture des aides est très partielle ! Nous demandons la commune de l'établissement aujourd'hui seulement pour calculer le versement transport. 

Pourquoi ? Car nous n'avons pas eu le temps ou la personne investie pour gérer les deux composantes de l'implémentation : avoir des donnéees exploitables des zones, et implémener les calculs d'exonération.

Nous n'avons pas d'indicateur de couverture en effet, nous sommes donc incapables de donner un chiffre sur la couverture. Je ne pense pas qu'un tel chiffre puisse exprimer toutes les dimensions du problème. 

En effet, nous pourrions ajouter une mention sur les aides et pourquoi pas faire un lien par exemple vers aides-entreprises.fr. Une référence vers ce site te semble-t-elle suffisante dans un premier temps ?

**Comment by juliencarnot** [comment]
Merci de ta confirmation rapide, je comprends les limites de vos moyens et ai manifestement un niveau d'attente trop élevé, mais j'imagine  que je ne dois pas être le seul à être dérouté par un simulateur produit par l'État qui n'indique pas clairement ce qu'il ne prend pas en compte. S'agissant d'exonérations sur les charges, je crois qu'il faut trouver une formulation très explicite et la plus exhaustive possible sur ce pour quoi on peut espérer un résultat juste et ce qu'il faut dénicher/vérifier ailleurs (ce qui constitue le backlog du simulateur en quelques sortes).

Je découvre tout juste aides-entreprises.fr et suis dépité de ne pas l'avoir trouvé plus tôt à travers mes recherches ou des liens depuis d'autres sites officiels... Cela me rappelle d'autres exemples sur [mpal](https://github.com/betagouv/mpal/issues/46) ou sur mes aides (cf. un échange mail avec Matti sur mesdroitssociaux.gouv.fr vs. mes-aides.gouv.fr ) dans lesquels j'ai été surpris par le manque de liens entre les différentes applications existantes ou nouvelles de l'État plateforme...

**Comment by laem** [comment]
Comme tu le notes bien, malheureusement nos moyens sont limités, ainsi se lancer dans la liste des dispositifs spécifiques que l'on n'implémente pas nous prendrait tout le temps qu'on utilise aujourd'hui pour implémenter ce qui est commun et aide déjà beaucoup. De façon similaire, il y a les conventions collectives qui souvent alourdissent le "coût", et qu'il ne sert à rien d'implémenter sans support de base solide.

 Je vais améliorer l'avertissement et faire le lien avec aides-entreprises.fr.

**Comment by laem** [comment]
@guillett

**Comment by juliencarnot** [comment]
En y regardant de plus près, je ne trouve pas d'infos sur l'exonération ZRR sur aides-entreprises.fr, par contre il y a des infos plus ou moins détaillées et à jour sur:

- https://les-aides.fr/fiche/a5VgGWNG2e3M/urssaf/zrr-zone-de-revitalisation-rurale-exoneration-de-cotisations-sociales.html
- https://www.afecreation.fr/pid538/exoneration-sociales-en-zrr.html
- https://www.urssaf.fr/portail/home/employeur/beneficier-dune-exoneration/exonerations-ou-aides-a-caracter/les-zones-de-revitalisation-rura.html
- https://www.pole-emploi.fr/employeur/l-exoneration-de-charges-sociales-dans-les-zones-defavorisees-@/article.jspz?id=60611

Difficile de dire sur quel site il faudrait renvoyer le visiteur sans connaitre son besoin d'info...

**Comment by laem** [comment]
Le nombre de communes est conséquent ! https://www.legifrance.gouv.fr/affichTexte.do?cidTexte=JORFTEXT000034298773
Edit : en effet, ZRR = commune sous la médiane de population et de densité, donc beaucoup d'entre elles.

**Comment by laem** [comment]
Ce sujet révèle bien le malaise   :cry:   https://www.compta-online.com/zone-zrr-reduction-fillon-t35871

**Comment by laem** [comment]
Je note ces conditions un peu particulières :

- vous n’avez pas procédé à un licenciement pour motif économique dans les 12 mois précédant l’embauche. 

- les salariés pour lesquels l’employeur bénéficie d’une mesure non cumulable avec l’exonération à l’embauche en ZRR. Cette condition est à investiguer. Il semble donc que la zrr n'ait de l'intérêt que pour étendre la réduction générale au delà de 1.6 x le SMIC.

- seulement 12 mois d'aide.

**Cette page que je ne comprends pas encore semble déterminante** ... https://www.urssaf.fr/portail/home/employeur/beneficier-dune-exoneration/exonerations-ou-aides-a-caracter/les-zones-de-revitalisation-rura/conditions.html

**Comment by laem** [comment]
J'ai compris ! C'est un algorithme qui vérifie que l'embauche vient augmenter l'effectif de l'entreprise.
Qui permet notamment de vérifier que l'entreprise n'a pas prévu de bénéficier de cette réduction en réduisant de l'autre côté son effectif par divers moyen...
